src/main/fc/rc.c

```diff
@@ -545,12 +541,7 @@ static FAST_CODE uint8_t processRcSmoothingFilter(void)
 
         rcSmoothingData.derivativeFilterTypeSetting = rxConfig()->rc_smoothing_derivative_type;
         if (rxConfig()->rc_smoothing_derivative_type == RC_SMOOTHING_DERIVATIVE_AUTO) {
```

Comment:
Looks like this is not 'auto' any more - we can probably remove this option from the list, and default to `RC_SMOOTHING_DERIVATIVE_BIQUAD` instead.
This will give us some more reduction of complexity.

Reply:
thanks, yes.
think I've fixed that now